meters/impl/abb.go

```diff
@@ -28,10 +28,15 @@ func NewABBProducer() Producer {
 		CurrentL2: 0x5B0E,
 		CurrentL3: 0x5B10,
 
-		Power:   0x5B24, // Apparent Power
```

Comment:
this basically changed apparent to active. ok for me.

Reply:
a friend with an SDM said, that on SDM the Power readings are active too